### PR TITLE
corrrect getExponent spec and impl

### PR DESCRIPTION
**Pull request by manuelbarzi**

### CODE REVIEW

File: stuff/javascript/jasmine-standalone-2.8.0/spec/getExponentSpec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+        expect(result).toBe(expected);
+    });
+
+    it('should throw an error on no arguments', function() {
+        expect(function() {
+        	getExponent();
+        }).toThrowError();
+    });
+});
```
Comment (marcrg) [inline stuff/javascript/jasmine-standalone-2.8.0/spec/getExponentSpec.js:14]: The message "No new line at end of file" is not a critical problem. But please, check this link -> 

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

File: stuff/javascript/jasmine-standalone-2.8.0/src/getExponent.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function getExponent(a, b) {
+	var res = Math.pow(a, b);
+
+	if (isNaN(res)) 
+		throw Error('invalid conditions');
+
+    return res;
+}
```
Comment (marcrg) [inline stuff/javascript/jasmine-standalone-2.8.0/src/getExponent.js:8]: The message "No new line at end of file" is not a critical problem. But please, check this link ->

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

**Review by marcrg — DISMISSED** [review]
Please, review comments

File: stuff/javascript/jasmine-standalone-2.8.0/spec/getExponentSpec.js (hunk 1 above)
Comment (manuelbarzi) [reply stuff/javascript/jasmine-standalone-2.8.0/spec/getExponentSpec.js:14]: no pasa naaa marc... ta toho bien papi!

**Review by marcrg — APPROVED** [review]

**Pull request merged**